setup.py

```diff
@@ -9,11 +9,12 @@
 INSTALL_REQUIRES = [
     'boto3>=1.9.156',
     'celery>=4.3.0',
-    'compliance-checker==4.1.1',
+    'compliance-checker>=4.1.1',
     'jsonschema>=2.6.0',
     'paramiko>=2.6.0',
     'python-magic>=0.4.15',
     'tableschema>=1.19.4',
+    'tabulate>=0.8.2',
```

Comment:
This was originally in here, but was removed because it was also a requirement for compliance-checker 4.x (#193).
However, the checker removed that dependency in 5.x
I guess the lesson learned is that since the code in `aodncore` explicitly imports `tabulate`, it _should_ be listed in the requirements.